intro-to-rails/rails-forms.md

```diff
@@ -182,19 +184,126 @@ The above ERB code generates this HTML:
 
 In `BooksController#new`, give the book instance variable a default title.  Do you notice a change in the form?
 
-  <details>
-    <summary>Solution</summary>  
-    The form's title field should now display the text which was set in the controller.
-  </details>
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
 
-Next put in a `@book.save` to the controller method.  Do you notice any change in the resulting HTML?  Look at the HTML output in Chrome Developer Tools.
+### !challenge
 
-  <details>
-    <summary>Solution</summary>
-  
-Because the Book instance now already exists in the database with an id field, the form will now submit to the `book_path` i.e. "/books/:id" and should now submit a patch request instead of a post request.  Therefore you could simply copy and paste the form into the `edit.html.erb` file and the form would work perfectly... but that doesn't seem very dry...
+* type: multiple-choice
+* id: 01badb58-a899-4b1e-a712-c5fbbe91f15b
+* title: Rails Forms Instance Variable
+* points: 1
+* topics: rails, rails-forms
+
+##### !question
+
+In `BooksController#new`, give the book instance variable a default title.  
+
+```ruby
+def new
+  @book = Book.new
+  @book.title = "Great Title"
+end
+```
+
+Then reload the form in the browser.
+
+Do you notice a change in the form?
+
+##### !end-question
+
+##### !options
+
+* Nope, no changes
+* Yes the form's title field has "Great Title" in it.
+* Yes, the app crashes!
+* Yes, the HTML now submits a PATCH request instead of a POST request
+
+##### !end-options
+
+##### !answer
+
+* Yes the form's title field has "Great Title" in it.
+
+##### !end-answer
+
+<!-- other optional sections -->
+<!-- !hint - !end-hint (markdown, users can see after a failed attempt) -->
+<!-- !rubric - !end-rubric (markdown, instructors can see while scoring a checkpoint) -->
+##### !explanation
+
+The form's title field should now display the text which was set in the controller.  The title comes from the instance variable as we use the instance variable to fill in form fields from:
+
+```ruby
+<%= form_with model: @book, class: 'create-book' do |f| %>
+```
+
+##### !end-explanation
+
+### !end-challenge
+
+<!-- ======================= END CHALLENGE ======================= -->
+
+<!-- >>>>>>>>>>>>>>>>>>>>>> BEGIN CHALLENGE >>>>>>>>>>>>>>>>>>>>>> -->
+<!-- Replace everything in square brackets [] and remove brackets  -->
+
+### !challenge
+
+* type: multiple-choice
```

Comment:
Just a weird question I have:

GitHub all of a sudden started changing the colors on the formatted text (the comments, headers, and text look like they're being read as code blocks or something?), which usually for me is an indicator of a syntax error.


I don't ACTUALLY see a syntax error, and I rain `learn preview .` and everything showed up fine... but just in case, just wanted to leave a comment. Do you see anything fishy?

Reply:
Also, I just realized that `learn preview .` DOES say there's an error on the page.

Reply:
The error is a link that's not in the branch, that I've pulled in now. Also ERB confused the hell out of Github.